app/models/king.rb

```diff
@@ -1,15 +1,98 @@
 class King < Piece
+
+  def y_position
```

Comment:
@lasley Isn't this just setting a variable y_position to a number and not effecting what's in the db? Don't you have to call self.y_position to effect what is in the db?? Then save.